welcome/welcome-to-d.md

```diff
@@ -5,54 +5,69 @@ Bem-vindo ao tour interativo da *linguagem de programação D*.
 {{#dmanmobile}}
 
 Esta tour dá uma visão geral desta __poderosa__ e __expressiva__
-linguagem que compila diretamente para __eficiente__ código __nativo__ de máquina.
+linguagem que compila diretamente para código de máquina __nativo__
+__eficiente__.
 
 {{/dmanmobile}}
 
-### O que é D?
+### O que é o D?
 
-D é o culminar de _décadas de experiência implementando compiladores_
-para diversas linguagens e tem um grande número de
-[funcionalidades únicas](http://dlang.org/overview.html):
+D é o culminar de _décadas de experiência em implementação de compiladores_
+para multiplas linguagens. Esta linguagem tem um grande número de
+[características únicas](http://dlang.org/overview.html):
 
 {{#dmandesktop}}
 
-- Construção de _alto nível_ para uma modelagem poderosa
+- construção em _high level_ que proporciona um alto poder de modelação
 - _alta performance_, linguagem compilada
-- tipagem estática
-- interface direta para APIs do sistema operacional e hardware
+- suporta definições estáticas
+- interface direta com APIs do sistema operativo e hardware
 - tempo de compilação incrivelmente rápido
-- permite programação com memory-safe (SafeD)
-- _manutenível_, código _fácil de entender_
-- baixa curva de aprendizado (sintaxe semelhante a C, Java e outras linguagens)
-- compatível com interface binário em C
-- compatibilidade limitada com interface binário em C++
-- multi-paradigma (imperativo, estruturado, orientado a objetos, genérica, programação funcional pura, e mesmo assembly)
-- prevenção de erros embutido (contratos, testes uniários)
+- permite programação em modo memory-safe (SafeD)
+- _manutenção fácil_, código _fácil de entender_
+- baixa curva de aprendizagem (sintaxe semelhante a C, Java e outras linguagens)
+- compatível com a interface binária do C
+- compatibilidade limitada com a interface binária do C++
+- multi-paradigma (imperativa, estruturada, orientada a objetos, genérica, programação funcional pura, e mesmo assembly)
+- deteção de error automática (contratos, _unittests_)
 
 ... e muitas outras [funcionalidades](http://dlang.org/overview.html).
 
 {{/dmandesktop}}
 
 ### Sobre o tour
 
-Cada seção vem com um código-fonte de exemplo que pode ser modificado e usado
-para experimentar as funcionalides da linguagem D.
-Clique no botão run (ou `Ctrl-enter`) para compilar e executa-ló.
+Cada secção inclui um código exemplo que pode modificado e usado para
+exprimentar as características da linguagem D.
+Para compilar o código e executá-lo, basta clicar no botão "Run"
+(ou `Ctrl-enter`).
 
-### Contribuindo
+### Contribuições
 
-Este tour tem o [código aberto](https://github.com/dlang-tour) e ficamos felizes
+O tour da linguagem D é [_open source_](https://github.com/dlang-tour).
+Para contribuir basta abrir um _pull request_
 com pull requests para torná-lo melhor.
 
 ## {SourceCode}
 
 ```d
 import std.stdio;
 
-// Vamos começar!
 void main()
 {
+    // Vamos começãr!
```

Comment:
`começ**ã**r` @FFY00 Is it really right in European Portuguese?

Reply:
No. It's a typo.